.github/workflows/test.yaml

```diff
@@ -113,17 +113,25 @@ jobs:
         with:
           node-version: 12
 
+      - name: Set test specific DATABASE_URL
+        run: |
+          string=${{ env.DATABASE_URL_POSTGRES_TEMPLATE }}
+          search=database
+          replace=${{ github.job }}_${{ matrix.feature }}_${{ matrix.os }}_${{ matrix.engine }}
+          replaced=${string/$search/$replace}
+          echo "DATABASE_URL=$replaced" >> $GITHUB_ENV
```

Comment:
This is copied to each job, slightly adapted to use the correct name of the feature key, in this case `feature`. Sometimes it is called differently though.